docs/reference/api/drivers/ResetReason.md

```diff
@@ -0,0 +1,39 @@
+## ResetReason
+
+Use the ResetReason interface to access hardware reset reason registers in a portable fashion.
+
+### ResetReason class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/v5.8/mbed-os-api-doxy/classmbed_1_1_resetreason.html)
```

Comment:
This link doesn't work. Please fix it.

Reply:
The intended pages don't currently exist. Could you generate the docs pages to link to under https://os.mbed.com/docs/v5.7/feature-hal-spec-watchdog-doxy/modules.html

Reply:
Thanks for letting me know. I'll talk with @iriark01 about having these rendered.

Reply:
Sorry, have what rendered? Is it just a feature branch to add?

Reply:
This isn't actually a feature branch. This is a driver page. Can you please help me find the correct link here: https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/annotated.html

Reply:
This? https://os-doc-builder.test.mbed.com/docs/development/feature-hal-spec-watchdog-doxy/reset__reason__api_8h_source.html

Reply:
The doxygen documentation should exist in the modules section under drivers [https://os-doc-builder.test.mbed.com/docs/development/feature-hal-spec-watchdog-doxy/group__drivers.html](https://os-doc-builder.test.mbed.com/docs/development/feature-hal-spec-watchdog-doxy/group__drivers.html), but for some reason aren't being listed.

Reply:
I can't see it in the exclusion, but I'm never quite sure how that works

https://github.com/ARMmbed/mbed-os/blob/feature-watchdog/doxygen_options.json

Reply:
Ahh thanks, it looks like it might be the lack of DEVICE_RESET_REASON and DEVICE_WATCHDOG in the predefined section.

Reply:
@scartmell-arm is that something you can fix?

Reply:
https://github.com/ARMmbed/mbed-os/pull/6332